en/components/grids_templates/excel_style_filtering.md

```diff
@@ -450,6 +450,94 @@ In order to provide a custom loading template for the excel style filtering, we
 
 <div class="divider--half"></div>
 
+
+### External Excel Style filtering
+
+As you see at the demos above the default appearance of the Excel Style filtering dialog is inside the grid. So this dialog is only visible when configuring the fitlers. There is a way to make that dialog stay always visible - it can  be used outside of the grid as a standalone component. All you need to do is to add it in your html and set its `column` property.
+
+#### Demo
+
+@@if (igxName === 'IgxGrid') {
+<div class="sample-container loading" style="height:654px">
+    <iframe id="grid-sample-1-iframe" src='{environment:demosBaseUrl}/grid/grid-external-excel-style-filtering' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="grid-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+}
+
+@@if (igxName === 'IgxTreeGrid') {
+<div class="sample-container loading" style="height:658px">
+    <iframe id="grid-sample-1-iframe" src='{environment:demosBaseUrl}/tree-grid/tree-grid-external-excel-style-filtering' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="grid-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+}
+
+@@if (igxName === 'IgxHierarchicalGrid') {
+<div class="sample-container loading" style="height:658px">
+    <iframe id="grid-sample-1-iframe" src='{environment:demosBaseUrl}/hierarchical-grid/hierarchical-grid-external-excel-style-filtering' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<br/>
+<div>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="grid-sample-1-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+}
+
+
+<div class="divider--half"></div>
+
+#### Usage
+
+At the sample above a column should be selected first, in order to instruct the ExcelStyle filter dialog which grid's column to filtrate. You can filter, hide or show, pin or unpin given column via that dialog. All you need to do, is to create the excel style dialog and bind its column property to a grid's column:
```

Comment:
Replace that paragraph with something like 'In order to configure the Excel style filtering component, you should set its `column`(API link) property to one of the @@igComponent's columns. In the sample above, we have bound the `column`(API link) property to the `value` of an `IgxSelectComponent` that displays the @@igComponent's columns.'

Reply:
Not sure about the API link...